### ISSUE TITLE
关于padding部分代码的疑惑

### ISSUE THREAD

**Issue by oner0916** [easezyc/WSDM2022-PTUPCDR#12]
pos_seq = keras.preprocessing.sequence.pad_sequences(data.pos_seq.map(self.seq_extractor), maxlen=20, padding='post')
这个代码中直接将padding设为0，但是数据处理代码中并没有将iid进行加一操作，也就是pos_seq序列中存在id=0的item，那为什么padding也为0呢？
